### DISCUSSION TITLE
Cannot flash FOXEER F405V2 INAV

### DISCUSSION THREAD

**Discussion by cder105** [iNavFlight/inav#10115] [Q&A]
Hello, I'm recently looking for help with Foxeer F405 V2, i want to configurate my drone in Inav for Mission Planning. How can i flash firmware for this flight controller if in Inav is only for F405?  If flash with F405 then there is not such Gyro ICM42688-P....

**Comment by mike-tk** [comment]
Any problems adding it to inav? It have targets FOXEERF405V2(BetaFlight); F405V2(Ardupilot)

**Reply by MrD-RC** [reply]
You should ask Foxeer that question. INAV has a new hardware policy that needs to be followed.
